docs/new/posts/Percona Monitoring and Management/pmm-3.0.0-release.md

```diff
@@ -0,0 +1,82 @@
+
+- [MongoDB Router Summary](https://docs.percona.com/percona-monitoring-and-management/3/reference/dashboards/dashboard-mongodb-router-summary.html){:target="_blank"}: dashboard now generally available
+- [Tech Preview support for MongoDB 8.0](https://www.mongodb.com/docs/manual/release-notes/8.0/){:target="_blank"}: PMM now offers technical preview support for MongoDB 8.0 and Percona Server for MongoDB 8.0, with updated exporters to handle revised metrics structures and renamed metrics.
+- [Increased query length limit for MongoDB in QAN](https://docs.percona.com/percona-monitoring-and-management/3/release-notes/3.0.0.html#increased-query-length-limit-for-mongodb-in-qan){:target="_blank"}: PMM extends MongoDB query length limits to 4096 characters and MySQL SlowLog query IDs to 32 characters, improving Query Analytics accuracy for both databases.
+
+### Breaking changes and deprecations
+
+- [Oracle Enterprise Linux 9 images only](https://docs.percona.com/percona-monitoring-and-management/3/release-notes/3.0.0.html#oracle-enterprise-linux-9-images-only){:target="_blank"}: PMM 3 now exclusively uses Oracle Enterprise Linux 9 (EL9) as its base system for all images, replacing EL7 to ensure better security updates and compatibility with new technologies.
+
+- [DBaaS functionality moved to Percona Everest](https://www.percona.com/resources/percona-everest){:target="_blank"}: PMM 3 has deprecated its DBaaS functionality in favor of Percona Everest, a cloud-native database platform that offers infrastructure control and monitoring integration while eliminating cloud vendor lock-in.
+
+- [Integrated Alerting deprecated and APIs removed](https://docs.percona.com/percona-monitoring-and-management/3/alert/index.html){:target="_blank"}
+- [Breaking API changes including simplified feature toggles](https://docs.percona.com/percona-monitoring-and-management/3/release-notes/3.0.0.html#breaking-api-changes){:target="_blank"}: PMM's API has been overhauled with simplified identifiers, streamlined feature controls, consistent response fields, and consolidated management endpoints.
+
+- [Grafana Angular support to be discontinued](https://docs.percona.com/percona-monitoring-and-management/3/release-notes/3.0.0.html#grafana-angular-support-discontinuation){:target="_blank"}: PMM is proactively migrating plugins to newer technologies ahead of Grafana's Angular deprecation in version 12 (2025), with recommendations to review and update dashboard plugins.
+
+
+## Component upgrades
+- Grafana upgraded to 11.1.8
+- Node Exporter updated to 1.8.2
+- ClickHouse Datasource plugin updated
+- ClickHouse-go driver upgraded to version 2
+
+## Ready to install or migrate to PMM 3?
```

Comment:
Maybe this section should go up? You already have Get hands-on with PMM. And now again, at the bottom. It confuses

Reply:
moved, thanks!